langchain/utilities/openweathermap.py

```diff
@@ -0,0 +1,73 @@
+"""Util that calls OpenWeatherMap using PyOWM."""
+
+from typing import Any, Dict, Optional
+
+from pydantic import Extra, root_validator
+from pyowm import OWM

```

Comment:
lets not add this in the file, but treat as optional dependency so only do inside functions when needed

Reply:
I removed the import from the file and added it as an optional dependency